embulk-jruby-extension/src/main/java/org/jruby/util/ParsedValues.java

```diff
@@ -0,0 +1,406 @@
+package org.jruby.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.jruby.util.RubyDateParser.isDigit;
+import static org.jruby.util.RubyDateParser.isSign;
+import static org.jruby.util.RubyDateParser.toInt;
+
+public class ParsedValues
```

Comment:
I think putting ParsedValues to org.jruby.util package is not a good idea because the name is generic. Why don't you move this to RubyDateParser.ParsedValues?